aspnetcore/fundamentals/routing.md

```diff
@@ -263,9 +263,13 @@ URL patterns that attempt to capture a filename with an optional file extension
 
 You can use the `*` character as a prefix to a route parameter to bind to the rest of the URI. This is called a *catch-all* parameter. For example, `blog/{*slug}` matches any URI that starts with `/blog` and had any value following it (which is assigned to the `slug` route value). Catch-all parameters can also match the empty string.
 
+The catch-all parameter will escape all characters when the route is used to generate a URL, including path separator `/` characters. For example, the route `foo/{*path}` with `Url.Action(new { path = "my/path" })` generates `foo/my%2Fpath`. Note the escaped forward slash. To round-trip path separtor characters you can use the `**` route parameter prefix. The route `foo/{**path}` with `Url.Action(new { path = "my/path" })` generates `foo/my/path`.
```

Comment:
> `Url.Action(new { path = "my/path" })`

So I'm not sure if any MVC concepts are really introduced yet. I think most users are probably most familiar with MVC...  If you wanted to make this example for neutral you could just say something like:

"For example, the route ... with route values ... "